orca-core/src/test/groovy/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictionsSpec.groovy

```diff
@@ -0,0 +1,45 @@
+package com.netflix.spinnaker.orca.config
+
+import spock.lang.Specification
+
+class UserConfiguredUrlRestrictionsSpec extends Specification {
+
+  def "validateURI fails if URI is a rejected IP"() {
+    setup:
+    def urlRestrictions = new UserConfiguredUrlRestrictions.Builder()
+      .withRejectedIps(Arrays.asList("10.63.245.46"))
+      .build()
+
+    when:
+    urlRestrictions.validateURI("10.63.245.46")
```

Comment:
These tests are passing an IP address to `validateURI` but it is actually expecting a host name.

Reply:
I believe I will have to get rid of this spec altogether.